README.md

```diff
@@ -25,3 +25,15 @@ To run the tests:
 ```bash
 npm test
 ```
+
+## Test app
+
+This is very much a work in progress.
+
+To run the test app:
+
+```
+$ cd test/app
```

Comment:
If this is accepting file uploads rather than just running tests, can we put it somewhere other than the `test/` folder?